cmd/ctr/commands/run/run_unix.go

```diff
@@ -186,8 +186,9 @@ func NewContainer(ctx gocontext.Context, client *containerd.Client, context *cli
 				opts = append(opts,
 					oci.WithUserNamespace([]specs.LinuxIDMapping{uidMap}, []specs.LinuxIDMapping{gidMap}))
 				// use snapshotter opts or the remapped snapshot support to shift the filesystem
-				// currently the only snapshotter known to support the labels is fuse-overlayfs:
-				// https://github.com/AkihiroSuda/containerd-fuse-overlayfs
+				// currently the snapshotters known to support the labels are:
+				// fuse-overlayfs - https://github.com/containerd/fuse-overlayfs-snapshotter
+				// overlay - in case of idmapped mount points are supported by host kernel (Linux kernel 5.12)
```

Comment:
Actually, I have a question -- do we really need this change? I mean `cmd/ctr/commands/run/run_unix.go`, because all we change is a comment. I'd prefer to leave it as is for time being

cc @fuweid @AkihiroSuda

Reply:
Yes, comments should be updated too

Reply:
Done